Client/Frontend/Settings/FxAContentViewController.swift

```diff
@@ -128,7 +128,9 @@ class FxAContentViewController: SettingsContentViewController, WKScriptMessageHa
         helper.delegate = self
         helper.application(app, didReceiveAccountJSON: data)
 
-        LeanplumIntegration.sharedInstance.setUserAttributes(attributes: [UserAttributeKeyName.signedInSync.rawValue : profile.hasAccount()])
```

Comment:
Why was this change made? Doesn't this mean that if the user decides to stop using FxA, the attribute will never reset to `false` ?